frontend/src/components/pages/TranslationPage.tsx

```diff
@@ -78,12 +51,11 @@ const TranslationPage = () => {
   const [translatedStoryLines, setTranslatedStoryLines] = useState<StoryLine[]>(
     [],
   );
+  const [changedStoryLines, setChangedStoryLines] = useState<
```

Comment:
@jennchenn FYI - Emma is recording what lines are changed. This has overlap with your current ticket